logging/logging-loki-ocp.adoc

```diff
@@ -0,0 +1,132 @@
+:_content-type: ASSEMBLY
+:context: logging-loki
+[id="logging-loki-ocp_{context}"]
+= Setting up logging
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+An essential function of {product-title} is the collection and aggregation of logs from the environments and application pods it runs.
+
```

Comment:
@periklis  stored in chunks in object stores or locally on the filesystem.  loki-operator  doesn't support data on filesystem.  Shall we update it?